setup.ps1

```diff
@@ -0,0 +1,229 @@
+# Set the console color to Matrix theme and clear the console
+$host.UI.RawUI.BackgroundColor = "Black"
+$host.UI.RawUI.ForegroundColor = "Green"
```

Comment:
I'm not sure this will go well with everyone - if i have adjusted my terminal for "my colorblindness" - then you're making it unreadable for that user by ignoring their settings.


Now I don't necessarily mind changing the console color here ... but if it's changed - we should change it back to what it was (both in case of success, or in case of error / failure).
If it's not possible to do that reliably (especially in case of error) - then this is something we cannot have.

Reply:
Ok will check what I can do here.

Reply:
I decided to remove it